utils/dataset_download.py

```diff
@@ -0,0 +1,19 @@
+import os
+from kaggle.api.kaggle_api_extended import KaggleApi
+
+link = 'https://www.kaggle.com/datasets/parisrohan/credit-score-classification' #just for convinience, not important
+download_path = "./data"
+dataset_name = "parisrohan/credit-score-classification"
+
+def download_dataset():
+    """create 'data' folder if it doesnt exist"""
+    os.makedirs(download_path, exist_ok=True)
+    
+    """activating kaggle api"""
+    api = KaggleApi()
+    api.authenticate()
+    
+    """self-explainatory"""
+    api.dataset_download_files(dataset_name, path=download_path, unzip=True)
```

Comment:
Return at the end something like True or "Dataset downloaded"(and add type annotation).